tests/network/bindingplugin.go

```diff
@@ -87,7 +87,11 @@ var _ = SIGDescribe("[Serial]network binding plugin", Serial, decorators.NetCust
 			Expect(err).ToNot(HaveOccurred())
 
 			vmi.Namespace = vm.Namespace
-			vmi = libwait.WaitUntilVMIReady(vmi, console.LoginToAlpine)
+			warningsIgnoreList := []string{"unknown error encountered sending command SyncVMI: " +
```

Comment:
Can you please follow the passt [tests](https://github.com/kubevirt/kubevirt/blob/1d48ef152713e1f8c382a7e07d4f09c14055a8f1/tests/network/vmi_passt.go#L107).

IIRC, the `DeadlineExceeded` is not the only warning the may occur on startup.

```
vmi = libwait.WaitForSuccessfulVMIStart(vmi,
					libwait.WithFailOnWarnings(false),
					libwait.WithTimeout(180),
				)
```

Reply:
I can, but then we may mask warnings which we do not expect to have.
This is the only one I saw to appear.
It is also used on the SR-IOV tests.

Don't you think we should limit the ignore as much as possible?
We could adjust it if we see other warnings which we think are acceptable.

Reply:
I prefer to keep it consistent with the other passt tests. We can try to understand what are that exact expected warnings and fix it all over the passt tests in a separate PR.

Reply:
OK, done.